content/applications/productivity/calendar/outlook.rst

```diff
@@ -41,37 +69,71 @@ URI (URL) followed by `/microsoft_account/authentication`.
 For more information on the restrictions and limitations of URIs, `check this page <https://docs.
 microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
 
-Regarding the application credentials, the user *must* add a client secret, which allows Odoo to
-authenticate itself, requiring no interaction from the user's side. :guilabel:`Certificates` are
-optional.
+Finally, click :guilabel:`Register` to complete the application registration. The
```

Comment:
Can you please specify *where* this is taking place? I'm assuming back on the app registration page, and not in Odoo? Because, up to this point, you've just guided the reader to the web.base.url page in the System Parameters of Odoo, and they be curious if this 'Register' button is *there*...So, if you could add a quick elaboration here, that would be super helpful. Thanks!